addons/website_form/views/website_form_templates.xml

```diff
@@ -1,140 +1,184 @@
-                                        </div>
-                                    </div>
-                                </div>
-                                <div class="form-group col-12 s_website_form_field s_website_form_required" data-type="char" data-name="Field">
-                                    <div class="row s_col_no_resize s_col_no_bgcolor">
-                                        <label class="col-form-label col-sm-auto s_website_form_label" style="width: 200px" for="contact5">
-                                            <span class="s_website_form_label_content">Subject</span>
-                                            <span class="s_website_form_mark"> *</span>
-                                        </label>
-                                        <div class="col-sm">
-                                            <input id="contact5" type="text" class="form-control s_website_form_input" name="subject" required=""/>
-                                        </div>
-                                    </div>
-                                </div>
-                                <div class="form-group col-12 s_website_form_field s_website_form_custom s_website_form_required" data-type="text" data-name="Field">
-                                    <div class="row s_col_no_resize s_col_no_bgcolor">
-                                        <label class="col-form-label col-sm-auto s_website_form_label" style="width: 200px" for="contact6">
-                                            <span class="s_website_form_label_content">Your Question</span>
-                                        </label>
-                                        <div class="col-sm">
-                                            <textarea id="contact6" class="form-control s_website_form_input" name="Description" required=""></textarea>
-                                        </div>
-                                    </div>
-                                </div>
-                                <div class="form-group col-12 s_website_form_field s_website_form_dnone">
-                                    <div class="row s_col_no_resize s_col_no_bgcolor">
-                                        <label class="col-form-label col-sm-auto s_website_form_label" style="width: 200px" for="contact7">
-                                            <span class="s_website_form_label_content">Email To</span>
-                                        </label>
-                                        <div class="col-sm">
-                                            <input id="contact7" type="hidden" class="form-control s_website_form_input" name="email_to"/>
-                                        </div>
+    <template id="contactus" name="Contact Us">
+        <t t-call="website.layout">
+            <div id="wrap">
+                <div class="oe_structure">
```

Comment:
Is is two nested div in a normal page? I had in mind that the `#wrap` itself was the oe_structure